noetic/distribution.yaml

```
@@ -9651,6 +9651,11 @@ repositories:
       url: https://github.com/botsync/volta.git
       version: noetic-devel
     status: maintained
+  vosk:
+    doc:
+      type: git
+      url: https://github.com/alphacep/ros-vosk
```

I started CI for this PR, but I know that its going to fail here. The repo needs .git attached. 
```suggestion
      url: https://github.com/alphacep/ros-vosk.git
```

---

Thank you!